### PR TITLE
Add an example of getting text input from a RemoteInputIntentHelper

### PR DESCRIPTION
**Pull request by JohnNichol**
Provide an example of getting text input from the user using RemoteInputIntentHelper. At the moment this is the only reliable way of getting text input as there are known issues with the foundation compose BasicTextField

Screenshot_20220506_135119

Screenshot_20220506_135137
.

### CODE REVIEW

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt [hunk 1]
```diff
@@ -143,5 +154,45 @@ fun UserInputComponentsScreen(
...
+            Chip(
+                onClick = {
+                    val intent: Intent = RemoteInputIntentHelper.createActionRemoteInputIntent()
+                    val remoteInputs: List<RemoteInput> = listOf(
+                        RemoteInput.Builder("input_text")
+                            .setLabel(label)
+                            .build()
+                    )
+
+                    RemoteInputIntentHelper.putRemoteInputsExtra(intent, remoteInputs)
```
Comment (yschimke) [inline ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt:179]: TIOLI: Might be good to extract this to a val, cleans up this event handler.

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt [hunk 2]
```diff
@@ -143,5 +154,45 @@ fun UserInputComponentsScreen(
...
+                        textForUserInput = newInputText as String
+                    }
+                }
+            val label = stringResource(R.string.manual_text_entry_label)
+            Chip(
+                onClick = {
+                    val intent: Intent = RemoteInputIntentHelper.createActionRemoteInputIntent()
+                    val remoteInputs: List<RemoteInput> = listOf(
+                        RemoteInput.Builder("input_text")
+                            .setLabel(label)
```
Comment (yschimke) [inline ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt:175]: Maybe show 

```
                .wearableExtender {
                      setEmojisAllowed(true)
                    setInputActionType(EditorInfo.IME_ACTION_DONE)
                }
```

File: ComposeAdvanced/app/build.gradle [hunk 3]
```diff
@@ -128,6 +128,7 @@ dependencies {
     // Horologist
     implementation libs.horologist.composables
     implementation libs.horologist.compose.layout
+    implementation 'androidx.wear:wear-input:1.1.0'
```
Comment (yschimke) [inline ComposeAdvanced/app/build.gradle:131]:
```suggestion
    implementation libs.wear.input
```

**Review by yschimke — APPROVED** [review]

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt (hunk 2 above)
Comment (shumelchyk) [reply ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt:175]: +1, was also going to suggest it

**Review by shumelchyk — APPROVED** [review]

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt (hunk 1 above)
Comment (JohnNichol) [reply ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt]: Done

File: ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt (hunk 2 above)
Comment (JohnNichol) [reply ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/userinput/UserInputComponentsScreen.kt]: As discussed added a dependency on the Wear-input 1.2.0-alpha02 library and added wearableExtender

**Comment by yschimke** [comment]
LGTM

**Pull request merged**